getting-started edits (#136)
withastro/starlight: docs/src/content/docs/getting-started.mdx

:::caution[Work in progress]
Starlight is in early development.

> If you find something that's not working, [open an issue on GitHub](https://github.com/withastro/starlight/issues/new/choose) or let us know on [Discord](https://astro.build/chat).

Reviewer: ```suggestion
If you find something that’s not working, [open an issue on GitHub](https://github.com/withastro/starlight/issues/new/choose) or let us know on [Discord](https://astro.build/chat).
```

## Creating content with Starlight

Starlight is ready for you to add new content, or bring your existing files!

### File formats

> Starlight supports authoring content in Markdown and MDX. (There is experimental support for Markdoc available via the [Astro Markdoc integration](https://docs.astro.build/en/guides/integrations-guide/markdoc/).)

Reviewer: Suggestion to rephrase slightly:

```suggestion
Starlight supports authoring content in Markdown and MDX. (You can add support for Markdoc by adding the experimental [Astro Markdoc integration](https://docs.astro.build/en/guides/integrations-guide/markdoc/).)
```

Reviewer: If you like this edit (or this + tweaks of your own), this PRs all yours to merge!
